core/common/src/main/java/alluxio/wire/Configuration.java

```diff
@@ -131,6 +159,8 @@ private Configuration(GetConfigurationPResponse conf) {
 
     mClusterConfHash = conf.getClusterConfigHash();
     mPathConfHash = conf.getPathConfigHash();
+    mClusterConfLastUpdateTime = conf.getClusterConfigLastUpdateTime();
+    mPathConfLastUpdateTime = conf.getPathConfigLastUpdateTime();
```

Comment:
need to check if the response has those fields, for compatibility?

Reply:
also what does "update time" mean? Does it mean the "update time in this process", or the "update time on the master"?

Reply:
My point is, if the timestamp means "the timestamp that some changes are applied to THIS process", then the TS can be System.now(). Otherwise you rely on the server side to give you a timestamp, then you will be forced to consider compatibility, i.e. what if the server response has no such field.

Reply:
@maobaolong Test whether NPE

Reply:
@jiacheliu3 

> also what does "update time" mean? Does it mean the "update time in this process", or the "update time on the master"?

Both cluster conf update time and pathconf update time are comes from master.